packages/themes/src/components/radio.ts

```diff
@@ -0,0 +1,68 @@
+import { YoruStyleProperties } from '@yoru-ui/core';
+
+const baseStyle: YoruStyleProperties = {
+  display: 'inline-flex',
+  cursor: 'pointer',
+  alignItems: 'center',
+  marginRight: '8px',
+  '& > input': {
+    position: 'absolute',
+    width: '1px',
+    height: '1px',
+    padding: 0,
+    margin: '-1px',
+    overflow: 'hidden',
+    witeSpace: 'nowrap',
+    border: '0px',
+    '&:checked + .yoru-radio__control': {
+      border: 'none',
+      '&:after': {
+        content: '""',
+        display: 'block',
+        position: 'absolute',
+        left: '50%',
+        top: '50%',
+        transform: 'translate(-50%, -50%)',
+        width: '40%',
+        height: '40%',
+        borderRadius: '50%',
+      },
+    },
+  },
+  '& .yoru-radio__control': {
+    transition: '0.4s',
+    position: 'relative',
+    boxSizing: 'border-box',
+    border: '1px solid',
+    borderColor: 'gray.400',
+    borderRadius: '100%',
+    marginRight: '8px',
+  },
+};
+
+const sizes = {
+  sm: {
```

Comment:
maybe you can write like this for sizes:

```
sm: {
    '.yoru-radio': {
      '&__control': {
        height: '12px',
        width: '12px',
      },
      '&__label': {
        fontSize: label size,
      },
    },
  },
```
**and you can put any CSS that nested from .yoru-radio**

Well, I see in the storybook that our radio label font size does not change when the radio size is changed. did you forget to add the label font size or just like this?

Reply:
Yes font size still the same, reference from chakra.